### PR TITLE
Add split learning PELoss defense and DcorLoss defense benchmark.(IJC…

### PR DESCRIPTION
**Pull request by bear-zd**
**Type of change**

- [x] Add new papers (Please tell us why you think this paper is awesome!)
- [x] Fix the category of an existing paper/papers (Please tell us the reasons)
- [ ] Add a new tool/primitive/application with a new markdown page (Thank you! Also, please tell us more about this awesome thing!)

**Description**

## New defense method: PE Loss and Dcor Loss

Added PE loss as a defense method against LIA (Label Inference Attacks) and implemented the Dcor variant of the NoPeek method mentioned in the paper as a baseline defense algorithm. Provided design files for loss-based LIA defense methods. Testing shows compatibility with existing LIA methods and produces effective results.

PE Loss (Potential Energy Loss) and Dcor Loss (Distance Correlation Loss) are two loss functions designed to defend against label inference attacks. PE Loss counters clustering-based LIA attack methods by adjusting the spatial positioning of features through potential energy loss regulation, published in IJCAI 2024. Paper: [Protecting Split Learning by Potential Energy Loss](https://www.ijcai.org/proceedings/2024/0618.pdf)

Local testing was conducted using sim lia as the attack method, with PE loss and Dcor loss as defense methods
image

Partial results show the advance of the defense method. Otherwise, adjusting the hyperparameter alpha value affects defense effectiveness - higher alpha values provide better defense but may impact classification performance.

Command can be run as follows:

```bash
python benchmark_examples/autoattack/main.py mnist resnet18 sim_lia peloss
python benchmark_examples/autoattack/main.py mnist resnet18 sim_lia dcorloss
```

## Small fix of sim lia

Normalizing the feature before k-means will get better attack metrics.

### CODE REVIEW

**Review by da-niao-dan — APPROVED** [review]
lgtm
